main/src/com/google/refine/commands/cell/SplitMultiValueCellsCommand.java

```diff
@@ -84,6 +84,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
                                                                          fieldLengths);
                 Process process = op.createProcess(project, new Properties());
                 
+                performProcessAndRespond(request, response, project, process);
+            } else if ("cases".equals(mode)) {
+                AbstractOperation op = new MultiValuedCellSplitOperation(columnName, 
+                                                                         keyColumnName,
+                                                                         true);
+                Process process = op.createProcess(project, new Properties());
+
+                performProcessAndRespond(request, response, project, process);
```

Comment:
Instead of duplicating the two lines `Process process = op.createProcess(project, new Properties());` and `performProcessAndRespond(request, response, project, process);` in each case of the `if`/`else` block, can we move them outside so that they are shared for all blocks? (The original code already had that duplication, it just feels more painful now that we have more cases)

Reply:
@wetneb Yes, I should do that.